### PR TITLE
add doctest for jump_search.py

### PR DESCRIPTION
**Pull request by ghost**
Updates #169

### CODE REVIEW

**Comment by atin** [comment]
@cyberwizard1001 your review please

**Comment by ming-tsai** [comment]
Hi @sridhar-5,
Could you help me to check this PR?
Because @cyberwizard1001 is not active.

**Comment by sridhar-5** [comment]
> Hi @sridhar-5,
> Could you help me to check this PR?
> Because @cyberwizard1001 is not active.

Sure. i'd love to.

File: algorithms/Python/searching/jump_search.py [hunk 1]
```diff
@@ -37,13 +42,11 @@ def check_sort(test: list) -> bool:
 
 
 if __name__ == "__main__":
-    arr = [0, 1, 2, 8, 13, 17, 19, 25, 31, 32, 42]
-    target = 25
     if check_sort(arr):
```
Comment (sridhar-5) [inline algorithms/Python/searching/jump_search.py:46]: can you please elaborate on where is the check_sort() function please?

**Comment by cyberwizard1001** [comment]
Ah sorry, I didn't receive the mail. Will keep an eye from hereon

**Review by sridhar-5 — CHANGES_REQUESTED** [review]
Nice Work. Code looks good to me. But the array and target can be taken  from the user as input to make this more user friendly.

**Comment by atin** [comment]
Yeah but I think the main aim of the program is to demonstrate the algorithm and also if we take the input from user the doctests won't work.

**Comment by sridhar-5** [comment]
well in that case. this is fine and can be merged.

**Review by sridhar-5 — APPROVED** [review]
code look good to me. @ming-tsai this pull request is good enough to be merged.

**Comment by sridhar-5** [comment]
Nice work, @atin .

**Comment by atin** [comment]
@ming-tsai can you merge this?

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**